crates/polars-io/src/parquet/write.rs

```diff
@@ -195,6 +195,13 @@ where
         if n_splits > 0 {
             *df = accumulate_dataframes_vertical_unchecked(split_df(df, n_splits)?);
```

Comment:
Ai.. that's painful. Can you do this? 

```rust
let mut batched = self.batched(&df.schema())?;

// we must explicitly loop here, as we cannot get multiple null column chunks
for df in split_df(df, n_splits)? {
   batched.write_batch(df)?;
}
return batched.finish()
```

I think we might stop that optimization btw, as it will hurt us in the streaming engine as well.

Reply:
Yeap, sounds good.